lms/resources/_js_config/__init__.py

```diff
@@ -285,7 +285,12 @@ def enable_dashboard_mode(self, token_lifetime_seconds: int) -> None:
                         students=self._to_frontend_template("api.dashboard.students"),
                         assignment_grades_sync=self._to_frontend_template(
                             "api.dashboard.assignments.grading.sync"
-                        ),
+                        )
+                        if self._lti_user
```

Comment:
I mention using `is_staff` in the FE to show or not the button. This would make that superfluous in practice but maybe it makes sense to have both checks.